### PR TITLE
Fix UpdateResults Test Failures

### PR DESCRIPTION
**Pull request by khansaad**
This PR fixes the pending negative test failures of the updateResults API. 

- Added validation for the null and blank container images
- Skipped tests for the aggregation_info variables
- Skipped tests for invalid container name and container_image name

### CODE REVIEW

File: tests/scripts/remote_monitoring_tests/helpers/utils.py [hunk 1]
```diff
@@ -138,43 +138,20 @@
     "container_image_name": "kruize/tfb-qrh:1.13.2.F_et17",
     "container_name": "tfb-server",
     "cpuRequest_name": "cpuRequest",
-    "cpuRequest_sum": 4.4,
-    "cpuRequest_avg": 1.1,
     "cpuRequest_format": "cores",
     "cpuLimit_name": "cpuLimit",
```
Comment (chandrams) [inline tests/scripts/remote_monitoring_tests/helpers/utils.py:142]: You should not remove cpu and memory usage values as these are used in other tests

File: tests/scripts/remote_monitoring_tests/helpers/utils.py [hunk 2]
```diff
@@ -190,6 +196,12 @@ def generate_test_data(csvfile, test_data, api_name):
         for key in test_data:
             for t in test_type:
                 data = []
+                # skip checking the invalid container name and container image name
+                if (key == "container_name" or key == "container_image_name") and t == "invalid":
+                    continue
+                #  skip checking the aggregation info values
+                if key in aggr_info_keys_to_skip:
```
Comment (chandrams) [inline tests/scripts/remote_monitoring_tests/helpers/utils.py:203]: Also add a condition t == "null" here to skip only these tests

Comment (chandrams) [reply tests/scripts/remote_monitoring_tests/helpers/utils.py]: Tested these looks fine

**Review by chandrams — APPROVED** [review]
LGTM

File: src/main/java/com/autotune/analyzer/serviceObjects/verification/validators/KubernetesElementsValidator.java [hunk 3]
```diff
@@ -93,6 +96,18 @@ public boolean isValid(UpdateResultsAPIObject updateResultsAPIObject, Constraint
                         ));
             }
 
+            // Validate blank or null container names and image names
+            List<ContainerData> resultContainers = resultData.getKubernetes_objects().get(0).getContainerDataMap().values().stream().toList();
```
Comment (msvinaykumar) [inline src/main/java/com/autotune/analyzer/serviceObjects/verification/validators/KubernetesElementsValidator.java:100]: any chance of index exception

**Review by chandrams — APPROVED** [review]
LGTM

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**